sync-core/src/test/java/com/cloudant/http/HttpTest.java

```diff
@@ -119,4 +122,52 @@ public void testReadBeforeExecute() throws IOException {
         Assert.assertEquals(bis.available(), data.getBytes().length);
         client.deleteDb();
     }
+
+
+    //NOTE: This test doesn't work with specified couch servers,
```

Comment:
Yeah, there already was a way of skipping, but I've updated it to use `Assume.false` in 9b86a41